packages/ses/src/tame-function-constructors.js

```diff
@@ -126,11 +126,17 @@ export default function tameFunctionConstructors() {
     '%InertAsyncFunction%',
     '(async function(){})',
   );
-  repairFunction(
-    'AsyncGeneratorFunction',
-    '%InertAsyncGeneratorFunction%',
-    '(async function*(){})',
-  );
+
+  try {
+    new FERAL_FUNCTION('async function* AsyncGeneratorFunctionInstance() {}')();
```

Comment:
Perhaps we can move this construction to `commons.js` and use `if (AsyncGeneratorFunctionInstance)` throughout instead of `try`.

Reply:
added in https://github.com/endojs/endo/pull/2334/commits/1f536cc7a93d182037b4395be8ada2b877acc24c looks good to use, feel free to suggest better wording if too verbose

Reply:
added in https://github.com/endojs/endo/pull/2334/commits/4b1b5e672477a7305e97601fcad26d48c186d141 looks good to use, feel free to suggest better wording if too verbose

Reply:
the only minor downside i see with re-using the `commons.js` construction, is the console info msgs are now more generic
`console.info('skipping async generators');`

whereas currently they're more verbose
`console.info('getAnonymousIntrinsics: skipping async generators');`
`console.info('tameFunctionConstructors: skipping async generators');`